oat-sa/extension-tao-testqti
# fix: TR-2486.Rewrite basic styles

Related to: https://oat-sa.atlassian.net/browse/TR-2486

mexcbt package contains some specific styles which were not take into account while developing task in terms of `tao-community`. This fix rewrite some theme styles to be aligned with design.

How to test:
1. setup https://github.com/oat-sa/mexcbt-package
2. switch tao-test-runner-qti-fe repo to branch related with https://github.com/oat-sa/tao-test-runner-qti-fe/pull/445 PR
3. check that panel work as previous

For setup you can use composer, which i used in Playground env: 
[composer.zip](https://github.com/oat-sa/tao-test-runner-qti-fe/files/7552201/composer.zip)
